src/components/WwwFrame/TheHeader.vue

```diff
@@ -426,6 +426,8 @@ export default {
 						return preFetchAll(matched, this.apollo, {
 							route: this.$route,
 							kvAuth0: this.kvAuth0,
+						}).catch(error => {
+							this.$router.push(error);
```

Comment:
We should check that the error is actually a route before sending it to router.push, either by checking `error instanceof Error` or `error.url`

Reply:
Added a check for error.path.